### ISSUE TITLE
Unnecessary space with "Group word tokens in a single menu"

### ISSUE THREAD

**Issue by GoogleCodeExporter** [legege/searchwp#81]
```
What steps will reproduce the problem?
1. Enable "Group word tokens in a single menu"
2. insert a long text into search box
3. resize search box to make it larger

What is the expected output? What do you see instead?
- See all / more text inside the search box but it shows unnecessary white 
space which even becomes more and more

What version of the product are you using? On what operating system?
- * (latest stable + svn tested) with FF 12 on Windows 7 x86


I've fixed it with attached .diff for searchbox.css
Basically it adds "max-width:20px;" for "*[tokensdisplaymode="3"] 
.searchbox-embedded-actions-layer" I still don't know what's causing this 
"dynamic" width
```

Original issue reported on code.google.com by `white06t...@gmail.com` on 27 Mar 2012 at 2:00
- Merged into: #86

Attachments:
- [searchbox.css.diff](https://storage.googleapis.com/google-code-attachments/searchwp/issue-81/comment-0/searchbox.css.diff)
